tests/utils/test_file_utils.py

```diff
@@ -99,13 +99,20 @@ def test_file_not_found(self):
         with self.assertRaisesRegex(EntryNotFoundError, "404 Client Error"):
             _ = get_from_cache(url)
 
-    @unittest.skip("Temp bug in the Hub not returning RepoNotFound errors.")
-    def test_model_not_found(self):
-        # Invalid model file.
+    def test_model_not_found_not_authenticated(self):
+        # Invalid model id.
         url = hf_bucket_url("bert-base", filename="pytorch_model.bin")
-        with self.assertRaisesRegex(RepositoryNotFoundError, "404 Client Error"):
+        with self.assertRaisesRegex(RepositoryNotFoundError, "401 Client Error"):
             _ = get_from_cache(url)
 
+    @unittest.skip("No authentication when testing against prod")
+    def test_model_not_found_authenticated(self):
+        # Invalid model id.
+        url = hf_bucket_url("bert-base", filename="pytorch_model.bin")
+        with self.assertRaisesRegex(RepositoryNotFoundError, "404 Client Error"):
+            _ = get_from_cache(url, use_auth_token="hf_sometoken")
+            # ^ TODO - if we decide to unskip this: use a real / functional token
+
```

Comment:
Skipping this test because we don't have an equivalent of `__DUMMY_TRANSFORMERS_USER__` in production 😕